packages/react-app/package.json

```diff
@@ -15,64 +15,48 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "type-check": "tsc --pretty --noEmit",
-    "format": "prettier --write .",
-    "lint": "eslint . --ext ts --ext tsx --ext js",
-    "test": "jest",
-    "test-all": "yarn lint && yarn type-check && yarn test"
+    "type-check": "tsc --pretty --noEmit --skipLibCheck --esModuleInterop --strict --jsx preserve",
+    "export": "next export",
+    "lint": "next lint",
+    "format": "prettier --write src"
+  },
+  "husky": {
```

Comment:
Husky is now configured correctly with yarn workspaces. It has a `pre-commit` hook, but since it only runs `yarn lint` for `react-app` it won't impact the `truffle` package at all.